mods/sp/weapons/superweapons.yaml

```diff
@@ -746,8 +781,9 @@ IronSavior:
 		Size: 2
 		InvalidTargets: Vehicle, Building, Wall
 		Delay: 15
-	Warhead@GetsOneShotedbySuperweapons: GrantExternalCondition
-		Range: 4c0
+	Warhead@GetsOneShotedbySuperweapons: SpreadDamage
+		Spread: 0c768
+		DamageTypes: Prone100Percent, TriggerProne, EnergyDeath
```

Comment:
Nerf this, its too much in contrast to the others superweapons, this one cleans the entire screen while the rest dont, its moslty that the secondary spheres have the same spread as the main one.

Reply:
Will nerf the second spheres, but air kill will have to remain the same because spread<768 will not affect air, which is strange.